### PR TITLE
OSDOCS#38575: Added inferFromVolume to creating VMs from container disks

### PR DESCRIPTION
**Pull request by jherrman**
Version(s):
4.17 and later

Issue:
https://issues.redhat.com/browse/CNV-38575

Links to docs preview:

https://84294--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/virtual_machines/creating_vms_custom/virt-creating-vms-from-container-disks.html
https://84294--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/virtual_machines/creating_vms_custom/virt-creating-vms-from-web-images.html
https://84294--ocpdocs-pr.netlify.app/openshift-rosa/latest/virt/virtual_machines/creating_vms_custom/virt-creating-vms-from-container-disks.html
https://84294--ocpdocs-pr.netlify.app/openshift-rosa/latest/virt/virtual_machines/creating_vms_custom/virt-creating-vms-from-web-images.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by dominikholler** [comment]
The usage on commandline improved significantly recently.
Let's work with @0xFelix to let this file reflect the current state.

**Review by 0xFelix — COMMENTED** [review]
Looking at it, I noticed the items 1-3 in the Procedure section are now also obsolete.

**Comment by jherrman** [comment]
@0xFelix , just to make sure, do you mean that creating a `Secret` manifest and making a config map for communicating to systems with self-signed certificates are no longer needed to create a VM from a container disk, or do you mean that different steps are needed for those than what the documentation currently describes?

Cheers,
J.

**Comment by 0xFelix** [comment]
They are not needed with the new VM manifest. The new manifests clones the `rhel9` DataSource, which is available in every cluster without specifying any additional credentials or certificates.

**Comment by jherrman** [comment]
Thank you for the clarification Felix! I've removed the steps, and I'm moving the ticket to QE review.

Thanks again for your help,
J.

File: modules/virt-creating-vm-import-cli.adoc [hunk 1]
```diff
@@ -77,73 +42,54 @@ apiVersion: kubevirt.io/v1
...
 spec:
   dataVolumeTemplates:
   - metadata:
       creationTimestamp: null
-      name: fedora-dv <2>
+      name: rhel-dv <2>
     spec:
+      sourceRef:
+        kind: DataSource
+        name: rhel9
```
Comment (RoniKishner) [inline modules/virt-creating-vm-import-cli.adoc:56]: The namespace might be needed here
```suggestion
      sourceRef:
        kind: DataSource
        name: rhel9
        namespace: openshift-virtualization-os-images
```

File: modules/virt-creating-vm-import-cli.adoc [hunk 2]
```diff
@@ -77,73 +42,54 @@ apiVersion: kubevirt.io/v1
...
-        resources:
-          requests:
-            memory: 1.5Gi
+        devices: {}
+        resources: {}
       terminationGracePeriodSeconds: 180
       volumes:
       - dataVolume:
-          name: fedora-dv
+          name: datavolumedisk1
```
Comment (RoniKishner) [inline modules/virt-creating-vm-import-cli.adoc:78]: Should be same as `dataVolumeTemplates` name
```suggestion
          name: rhel-dv <2>
```

**Review by RoniKishner — CHANGES_REQUESTED** [review]

**Review by snarayan-redhat — COMMENTED** [review]
LGTM!

**Review by RoniKishner — APPROVED** [review]

**Pull request merged**